options/locale/locale_zh-CN.ini

```diff
@@ -1229,6 +1229,8 @@ issues.new.add_reviewer_title=请求审核
 issues.choose.get_started=开始
 issues.choose.blank=默认模板
 issues.choose.blank_about=从默认模板创建一个工单。
+issues.choose.ignore_invalid_templates=已忽略无效的模板
+issues.choose.invalid_templates=有 %v 个无效的模板
```

Comment:
Only en-US translation is needed, others are maintained by crowdin. So these lines should be removed.

Reply:
Thanks for reminding me, I didn't know about this before, and these lines have been removed.